src/utils/arrays/index.ts

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: Apache-2.0
+
+export type TypedArray = Int8Array | Uint8Array | Uint8ClampedArray | Int16Array | Uint16Array | Int32Array | Uint32Array | Float32Array | Float64Array;
+
+export function equalArray(x: TypedArray, y: TypedArray): boolean {
+	return (x.length == y.length)
```

Comment:
```suggestion
	return (x.length === y.length)
```

Reply:
I really need to get used to this `===` business in JS ^^